tests/platform_tests/test_sfp.py

```diff
@@ -56,7 +59,17 @@ def parse_eeprom(output_lines):
             res[fields[0]] = fields[1].strip()
     return res
 
-def test_check_sfp_status_and_configure_sfp(duthosts, rand_one_dut_hostname, enum_frontend_asic_index, conn_graph_facts, tbinfo):
+def supports_sfputil_on_dut(duthost):
+    # check if duthost supports sfputils by checking the file in /usr/share/sonic/device/<platform>/plugins/sfputil.py
+    global supports_sfputil
+    supports_sfputil = True
+    sfputil_file_path = os.path.join("/usr/share/sonic/device", duthost.facts['platform'], "plugins/spfutil.py")
+    try:
+        duthost.shell("cat {}".format(sfputil_file_path))["stdout_lines"]
+    except Exception:
+        supports_sfputil = False
```

Comment:
sfputil.py (and all other old platform plugins) are deprecated in favor of the new sonic-platform API.

`sfpshow` will be used moving forward to display the latest SFP data gathered and stored in the State DB. However, I believe we should refactor sfputil to use the new platform API to provide us with a command line utility with which we can communicate directly with the transceivers if need be.

Therefore, I think we should not skip the sfputil tests if `plugins/spfutil.py` is not present. Instead, sfputil needs to be refactored to use the new platform API and we should expect that it should always work.

Reply:
OK - i can take the checks out, but until sfputils is refactored, the tests would fail on platforms that have not had sfputils refactored.  Going forward, is each vendor supposed to implement this refactored sfputil - or does it come from a common code for all vendors.

Reply:
I have committed changes to remove the checks for sfputils not supported

Reply:
@jleveque Should we change 'sfputil show eeprom' and 'sfputil show presence' to 'sfpshow'?

Reply:
> OK - i can take the checks out, but until sfputils is refactored, the tests would fail on platforms that have not had sfputils refactored. Going forward, is each vendor supposed to implement this refactored sfputil - or does it come from a common code for all vendors.

It will be a change in common code: [This file](https://github.com/Azure/sonic-utilities/blob/master/sfputil/main.py) needs to be refactored to utilize the vendor's sonic-platform package. I will take a look into this.

> @jleveque Should we change 'sfputil show eeprom' and 'sfputil show presence' to 'sfpshow'?

I think we need to test both sfputil and the new sfpshow, since they both serve different purposes.

Reply:
@sanmalho-git: I have opened a PR [here](https://github.com/Azure/sonic-utilities/pull/1421) to refactor sfputil so that it uses the sonic-platform package in lieu of the deprecated platform plugins. Can you please test and review the PR?

Reply:
Added sfpshow commands to test_sfp